src/app.js

```diff
@@ -162,7 +176,7 @@ export const start = args => {
       args = args.join('/');
     }
 
-    screen.spawn('git', ['log', args, '--color=always']);
+    screen.spawn('git', ['log', args, conf.get('logAlias')]);
```

Comment:
We should bind `conf.get()` to a variable up at the top and just pass the variable here. This way we can validate the options variable before passing it into git log.